.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+sudo: required
```

Comment:
I'm pretty sure you can copy the travis.yml file from any of our libraries:
https://github.com/blackbaud/skyux-core/blob/master/.travis.yml